Makefile

```diff
@@ -2,23 +2,34 @@ VERSION ?= $(shell git describe --tags --dirty)
 UNAME := $(shell uname)
 ARCH := $(shell uname -m)
 APP_NAME = embedded-cluster
-ADMIN_CONSOLE_CHART_REPO_OVERRIDE =
 ADMIN_CONSOLE_CHART_VERSION = 1.112.1
+ADMIN_CONSOLE_IMAGE_TAG = v1.112.1@sha256:c258cfe395b3aea134e6efde9e5ec5b19a1dadda4206905a2540ee2f9aa14e18
+ADMIN_CONSOLE_RQLITE_IMAGE_TAG = 8.26.7-r0@sha256:5100d28c8ccfed3ce35bc4b171770c26c5f98addebf42df97c5dcc64795eb28a
+ADMIN_CONSOLE_MIGRATIONS_IMAGE_TAG = v1.112.1@sha256:738fe80441500ace7d2fa1517d5482a8d60d8ec8d362c5c516b695852527d825
+ADMIN_CONSOLE_KURL_PROXY_IMAGE_TAG = v1.112.1@sha256:1f62caed3e9be4b5e3b4b6ff95e49381a80ce7392063f1b66c4de92e4c698a24
 ADMIN_CONSOLE_IMAGE_OVERRIDE =
 ADMIN_CONSOLE_MIGRATIONS_IMAGE_OVERRIDE =
 ADMIN_CONSOLE_KURL_PROXY_IMAGE_OVERRIDE =
+ADMIN_CONSOLE_CHART_REPO_OVERRIDE =
 EMBEDDED_OPERATOR_CHART_VERSION = 0.40.2
+EMBEDDED_OPERATOR_IMAGE_TAG = 0.40.2@sha256:8f464cc976210676bf4a75ec487b073e1902647449e4a945f195d51e94f0e4c8
+EMBEDDED_OPERATOR_PROXY_IMAGE_TAG = v0.13.1@sha256:18f12fc3a0085bbc303cfbd2ad1151127c1c959c4641e36027547b6b66eb4c92
+EMBEDDED_OPERATOR_UTILS_IMAGE_TAG = 1.36@sha256:50aa4698fa6262977cff89181b2664b99d8a56dbca847bf62f2ef04854597cf8
+EMBEDDED_OPERATOR_IMAGE_OVERRIDE =
 EMBEDDED_OPERATOR_BINARY_URL_OVERRIDE =
-EMBEDDED_OPERATOR_UTILS_IMAGE = busybox:1.36.1
-EMBEDDED_CLUSTER_OPERATOR_IMAGE_OVERRIDE =
 OPENEBS_CHART_VERSION = 4.1.0
-OPENEBS_UTILS_VERSION = 4.1.0
+OPENEBS_IMAGE_TAG = 4.1.0@sha256:7a9bb247229564c74b5c4981304bf614f1b0157797a52c6d882d5f1cb057b838
+OPENEBS_KUBECTL_IMAGE_TAG = 1.25.15@sha256:da199debb691621e0dc6954fefdaaf833900e58e28b96a4d805ff01ac397a979
+OPENEBS_UTILS_IMAGE_TAG = 4.1.0@sha256:a0f8fd9457388d5728e8e9c41f1cf5dc3b21ef8f00d7b019520101812ae602dc
 SEAWEEDFS_CHART_VERSION = 4.0.0
+SEAWEEDFS_IMAGE_TAG = 3.69@sha256:88eaab1398cd95b1e49f7210a36af389b4ebc896cccd2ef6f4baff3fa1b245fd
 REGISTRY_CHART_VERSION = 2.2.3
-REGISTRY_IMAGE_VERSION = 2.8.3
+REGISTRY_IMAGE_TAG = 2.8.1@sha256:85d4cd48754d4e8e4d4fe933bbde017d969e0e20775507add9faca39064bf9b0
```

Comment:
It seems the automation erroneously bumped this in the past. The latest registry chart (`v2.2.3`) ships with registry `2.8.1`.  I don't expect to see any problems once this is in I need to test an upgrade just to be sure.

Reply:
I would strongly prefer to keep using the 2.8.3 image version

Reply:
done.